PythonSourceGenerator/PythonSourceGenerator.csproj

```diff
@@ -12,7 +12,7 @@
             <PrivateAssets>all</PrivateAssets>
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
-        <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.10.0" />
+        <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.9.2" />

```

Comment:
@jaredpar why did the version go down in the changes?

Reply:
Specifically because I was debugging on the 8.0.200 SDK which has the 4.9.2 version of the compiler. Had to move the reference down so that the generator would load. 

In general though generators that load from NuPkg should pick the oldest version of the compiler that has the APIs you need. That maximizes the number of .NET SDK that your generator can load into. For example this generator would not be usable in .NET 8.0.100 SDK but probably could be by dropping down to 4.8.0.